### PR TITLE
Feature/nlp gui

### PR DESCRIPTION
**Pull request by puskuruk**
Screenshot_2020-03-03 Universal Data Tool
nlp-gui

### CODE REVIEW

**Comment by seveibar** [comment]
You can use this for the "labels" key

* https://collegeai.github.io/material-survey/?path=/story/dynamicmatrixquestion--text-type-columns

DeepinScreenshot_select-area_20200303111027

**Comment by seveibar** [comment]
Here's a link to the standard
https://github.com/UniversalDataTool/udt-format/blob/master/interfaces/text_entity_recognition.md

**Comment by puskuruk** [comment]
udt-nlp-gui-with-labels-

**Comment by seveibar** [comment]
nice!

File: src/components/ConfigureNLP/index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export default ({ iface, onChange }) => {    
+    return(
+        <Survey
+            noActions
+            variant="flat"
+            onQuestionChange={(questionId, newValue, answers) =>{
+                onChange({...iface, [questionId]: newValue})
+            }}
+            form={form}
```
Comment (seveibar) [inline src/components/ConfigureNLP/index.js]: needs to handle `defaultValues`, what if the user loads a file? Or if they navigate away and come back? See Computer Vision for an example for how to load default values (apologies in advance if it's WIP)

**Comment by puskuruk** [comment]
Default answers

Screenshot_2020-03-04 Universal Data Tool

**Pull request merged**